.github/workflows/playwright.yml

```diff
@@ -141,6 +141,36 @@ jobs:
         shell: bash
         working-directory: ${{ inputs.plugin-directory }}
 
+      - name: Get Playwright version
+        uses: actions/github-script@v7
+        id: version
+        with:
+          script: |
+            const workingDirectory = process.env.PLUGIN_DIRECTORY;
+            console.debug("Specified working directory:", workingDirectory);
+            if (workingDirectory) process.chdir(workingDirectory);
+            console.debug("Actual working directory:", process.cwd());
+            let version = "";
+            try {
+              version = require("@playwright/test/package.json").version;
+            } catch (error) {
+              console.log(error.message);
+            }
+            console.debug("Version:", version);
+            if (version) {
+              core.exportVariable("PLAYWRIGHT_VERSION", version);
+              core.setOutput("version", version);
```

Comment:
Do we need to export the variable for future steps, or does `core.setOutput` suffice? (and then we can use it in the next step as `${{ steps.version.outputs.version }}` rather than `${{ env.PLAYWRIGHT_VERSION }}`)